[Bug]: [Nightly]The ids of sealed segments was wrong after load balance

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 56e722f
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.3.0.dev21
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

The ids of sealed segments was wrong after load balance.
The case occasionally failed in kafka.
```
[2022-12-12T19:20:10.061Z] self = <test_utility.TestUtilityAdvanced object at 0x7fd7eb4e6700>
[2022-12-12T19:20:10.061Z] 
[2022-12-12T19:20:10.061Z]     @pytest.mark.tags(CaseLabel.L2)
[2022-12-12T19:20:10.061Z]     def test_load_balance_normal(self):
[2022-12-12T19:20:10.061Z]         """
[2022-12-12T19:20:10.061Z]         target: test load balance of collection
[2022-12-12T19:20:10.061Z]         method: init a collection and load balance
[2022-12-12T19:20:10.061Z]         expected: sealed_segment_ids is subset of des_sealed_segment_ids
[2022-12-12T19:20:10.061Z]         """
[2022-12-12T19:20:10.061Z]         # init a collection
[2022-12-12T19:20:10.061Z]         self._connect()
[2022-12-12T19:20:10.061Z]         querynode_num = len(MilvusSys().query_nodes)
[2022-12-12T19:20:10.061Z]         if querynode_num < 2:
[2022-12-12T19:20:10.061Z]             pytest.skip("skip load balance testcase when querynode number less than 2")
[2022-12-12T19:20:10.061Z]         c_name = cf.gen_unique_str(prefix)
[2022-12-12T19:20:10.061Z]         collection_w = self.init_collection_wrap(name=c_name)
[2022-12-12T19:20:10.061Z]         collection_w.create_index(default_field_name, default_index_params)
[2022-12-12T19:20:10.061Z]         ms = MilvusSys()
[2022-12-12T19:20:10.061Z]         nb = 3000
[2022-12-12T19:20:10.061Z]         df = cf.gen_default_dataframe_data(nb)
[2022-12-12T19:20:10.061Z]         collection_w.insert(df)
[2022-12-12T19:20:10.061Z]         # get sealed segments
[2022-12-12T19:20:10.061Z]         collection_w.num_entities
[2022-12-12T19:20:10.061Z]         # get growing segments
[2022-12-12T19:20:10.061Z]         collection_w.insert(df)
[2022-12-12T19:20:10.061Z]         collection_w.load()
[2022-12-12T19:20:10.061Z]         # prepare load balance params
[2022-12-12T19:20:10.061Z]         res, _ = self.utility_wrap.get_query_segment_info(c_name)
[2022-12-12T19:20:10.061Z]         segment_distribution = cf.get_segment_distribution(res)
[2022-12-12T19:20:10.061Z]         all_querynodes = [node["identifier"] for node in ms.query_nodes]
[2022-12-12T19:20:10.061Z]         assert len(all_querynodes) > 1
[2022-12-12T19:20:10.061Z]         all_querynodes = sorted(all_querynodes,
[2022-12-12T19:20:10.061Z]                                 key=lambda x: len(segment_distribution[x]["sealed"])
[2022-12-12T19:20:10.061Z]                                 if x in segment_distribution else 0, reverse=True)
[2022-12-12T19:20:10.061Z]         src_node_id = all_querynodes[0]
[2022-12-12T19:20:10.061Z]         des_node_ids = all_querynodes[1:]
[2022-12-12T19:20:10.061Z]         sealed_segment_ids = segment_distribution[src_node_id]["sealed"]
[2022-12-12T19:20:10.061Z]         # load balance
[2022-12-12T19:20:10.061Z]         self.utility_wrap.load_balance(collection_w.name, src_node_id, des_node_ids, sealed_segment_ids)
[2022-12-12T19:20:10.061Z]         # get segments distribution after load balance
[2022-12-12T19:20:10.061Z]         res, _ = self.utility_wrap.get_query_segment_info(c_name)
[2022-12-12T19:20:10.061Z]         segment_distribution = cf.get_segment_distribution(res)
[2022-12-12T19:20:10.061Z]         sealed_segment_ids_after_load_banalce = segment_distribution[src_node_id]["sealed"]
[2022-12-12T19:20:10.061Z]         # assert src node has no sealed segments
[2022-12-12T19:20:10.061Z] >       assert sealed_segment_ids_after_load_banalce == []
[2022-12-12T19:20:10.061Z] E       assert [438007954198702165] == []
[2022-12-12T19:20:10.061Z] E         Left contains one more item: 438007954198702165
[2022-12-12T19:20:10.061Z] E         Full diff:
[2022-12-12T19:20:10.061Z] E         - []
[2022-12-12T19:20:10.061Z] E         + [438007954198702165]
[2022-12-12T19:20:10.061Z] 
[2022-12-12T19:20:10.061Z] testcases/test_utility.py:1548: AssertionError
```

### Expected Behavior

pass

### Steps To Reproduce

_No response_

### Milvus Log

1. log: [artifacts-milvus-distributed-kafka-nightly-218-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/218/artifact/artifacts-milvus-distributed-kafka-nightly-218-pymilvus-e2e-logs.tar.gz)
2. collection: utility_HdbDzOuK
3. failed time: [2022-12-12T18:52:00.275Z] [gw0] [ 98%] FAILED testcases/test_utility.py::TestUtilityAdvanced::test_load_balance_normal

### Anything else?

_No response_

--- a/tests/python_client/testcases/test_utility.py
+++ b/tests/python_client/testcases/test_utility.py
@@ -1528,6 +1528,7 @@ class TestUtilityAdvanced(TestcaseBase):
         collection_w.insert(df)
         collection_w.load()
         # prepare load balance params
+        time.sleep(0.5)
         res, _ = self.utility_wrap.get_query_segment_info(c_name)
         segment_distribution = cf.get_segment_distribution(res)
         all_querynodes = [node["identifier"] for node in ms.query_nodes]

